ttnn/cpp/ttnn/graph/graph_processor.cpp

```diff
@@ -305,8 +305,27 @@ int GraphProcessor::add_tensor(const Tensor& t) {
     return tensor_counter;
 }
 
+namespace detail
+{
+    // hash buffer because address is not unique
```

Comment:
I removed this commit to avoid confusion. two unit tests on binary compiler interface will fail until I have that change in main and rebase to that commit.